### PR TITLE
added components: Mainpage, Categories. Mainpage renders Categories …

### PR DESCRIPTION
**Pull request by nilan01**
Adding the components: Mainpage, Categories.

Categories `fetches` all categories and displays them using `ListGroup`.
Categories is rendered on Mainpage.
Mainpage is rendered on App.js.

### CODE REVIEW

File: src/frontend/src/components/App.js [hunk 1]
```diff
@@ -2,14 +2,16 @@ import React from 'react';
...
 import 'bootstrap/dist/css/bootstrap.min.css';
 
 class App extends React.Component {
   render() {
     return (
       <div>
         <Header />
-        <Menuitems />
+        {/*<Menuitems />*/}
+        <Mainpage></Mainpage>
```
Comment (chrispinkney) [inline src/frontend/src/components/App.js:14]: Can you make this self closing like the ones above it?

File: src/frontend/src/components/Mainpage.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import Categories from './Categories';
+
+export default class Mainpage extends Component {
+  render() {
+    return (
+      <div>
+        <Categories></Categories>
```
Comment (chrispinkney) [inline src/frontend/src/components/Mainpage.js:8]: Here, too

File: src/frontend/src/components/App.js (hunk 1 above)
Comment (nilan01) [reply src/frontend/src/components/App.js]: Fixed, thank you.

File: src/frontend/src/components/Mainpage.js (hunk 2 above)
Comment (nilan01) [reply src/frontend/src/components/Mainpage.js]: Fixed, thank you.

**Review by chrispinkney — APPROVED** [review]

**Comment by chrispinkney** [comment]
LGTM

**Pull request merged**